src/redux/actions/cellSets/updateCellSetsClustering.js

```diff
@@ -8,7 +8,7 @@ const updateCellSetsClustering = (experimentId) => async (dispatch) => {
   try {
     dispatch({ type: CELL_SETS_CLUSTERING_UPDATED });
 
-    dispatch(loadCellSets(experimentId));
+    dispatch(loadCellSets(experimentId, true));
```

Comment:
why do we need to force reload?

Reply:
`updateCellSetsClustering` is used in `configureEmbedding` to update cell sets. This was set to force reload to handle the case of multiple users, with different browsers, looking at the same experiment at the same time. If one of the users recluster the cell sets (i.e. via configure embedding), the cell sets in the other users' browsers will also be forced to update. Otherwise, they would be looking at different cell sets, which may mess up cell set operations.

Reply:
so this is a bug that you are fixing with these changes?